contracts/liquidity_hub/epoch-manager/tests/epoch.rs

```diff
@@ -72,7 +71,7 @@ fn create_new_epoch_successfully() {
 //     // move time ahead but not enough so the epoch creation fails
 //     env.block.time = env.block.time.plus_nanos(86300);
 
-//     let msg = ExecuteMsg::CreateEpoch {};
+//     let msg = ExecuteMsg::CreateEpoch;
 //     let err = execute(deps.as_mut(), env, info, msg).unwrap_err();
 //     match err {
 //         ContractError::CurrentEpochNotExpired => {}
```

Comment:
check this commented code and remove it if not needed

Reply:
I enabled it again